subsys/nonsecure/Kconfig

```
@@ -0,0 +1,314 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if BUILD_WITH_TFM || IS_SPM
+menu "Trusted Execution Non-Secure"
+menu "Peripheral Secure mapping"
+
+config NRF_FPU_SECURE
+	bool "FPU configured as secure peripheral"
+	depends on CPU_HAS_FPU
+
+config CONFIG_NRF_DCNF_SECURE
```

a `CONFIG_` too much ?
```suggestion
config NRF_DCNF_SECURE
```